### PR TITLE
Awesome books ES6

### PR DESCRIPTION
**Pull request by marvin-nyalik**
In this branch:

- I used ES6 syntax in my code
- I used luxon library for dateTime
- I broke down my code to use different modules

Let me know of any required changes

### CODE REVIEW

File: modules/addBooks.js
Comment (LionRouge1) [inline modules/addBooks.js:1]: - [ ] Nice work implementing the functionality to add a book :clap: :sunglasses: However whenever a user refresh the page a new empty book is created, Kindly make sure a new book is create only when a user click the button "Add" in the "add new" section
Screenshot from 2023-06-05 13-40-07

File: index.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+const dateTime = DateTime.local();
+const formattedDateTime = dateTime.toLocaleString(DateTime.DATETIME_FULL);
+document.getElementById('date').innerHTML = formattedDateTime;
+
+(function onReload() {
+  if (localStorage.getItem('books')) {
+    const allBooks = JSON.parse(localStorage.getItem('books'));
+    displayBooks(allBooks);
+  }
```
Comment (LionRouge1) [inline index.js]: - [ ] Please refactor this functions to arrow functions as requested in the project requirement.
image

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @marvin-nyalik, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good job divided your code into reasonable modules :1st_place_medal:   
- Nice work using **arrow function** :+1: 
- All linters passed :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by addod19 — APPROVED** [review]
### Hi @marvin-nyalik, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [ ] Great job addressing the comments from the previous review :rocket: 
Screenshot from 2023-06-05 15-55-59



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**